pyrakoon/compat.py

```diff
@@ -1064,11 +1066,15 @@ def _get_connection(self, node_id):
 
 
 class _ClientConnection(object):
-    def __init__(self, address, cluster_id):
+    def __init__(self, address, cluster_id, timeout=0):
         self._address = address
         self._connected = False
         self._socket = None
         self._cluster_id = cluster_id
+        if (isinstance(timeout, int) and timeout > 0) or timeout is None:
```

Comment:
I would not do this. `_ClientConnection(....,timeout=3.5)` will be silently translated into something else.